src/core/embedders/FileEmbedder.ts

```diff
@@ -2,12 +2,14 @@ import PDFString from 'src/core/objects/PDFString';
 import PDFHexString from 'src/core/objects/PDFHexString';
 import PDFContext from 'src/core/PDFContext';
 import PDFRef from 'src/core/objects/PDFRef';
+import {LiteralObject} from 'src/core/PDFContext'
 
 export interface EmbeddedFileOptions {
   mimeType?: string;
   description?: string;
   creationDate?: Date;
   modificationDate?: Date;
+  additionalParams?: LiteralObject;
```

Comment:
Per https://github.com/Hopding/pdf-lib/pull/508#issuecomment-719968177 can we change this to:
```ts
/** 
 * From the ZUGFeRD specification, Supplement-A, section **2.2.2. Data Relationship**.
 * See:
 * * https://www.ferd-net.de/standards/zugferd-versionsarchiv/zugferd-2.1.html
 * * http://www.awv-net.de/updates/zugferd21/zugferd21en.zip
 */
export enum AFRelationship {
  Data = 'Data',
  Source = 'Source',
  Alternative = 'Alternative',
  Supplement = 'Supplement',
  Unspecified = 'Unspecified'
}

export interface EmbeddedFileOptions { 
  ...,
  afRelationship: AFRelationship;
}
```